scripts/postversion.sh

```diff
@@ -1,7 +1,7 @@
 #/bin/sh
 
 # Publish and push!
-git push;
-git push --tags;
-git push git@github.com:paypal/paypal-identity-components.git master;
-npm publish;
+# git push;
```

Comment:
Please back out all the changes we made in the `scripts` folder. Those were only for publishing to alpha.

Reply:
reverted